assets/src/block-templates/highlighted-cta/template.js

```diff
@@ -8,7 +8,6 @@ const template = ({
     {
       className: `block`,
       textColor: 'white',
-      backgroundColor: 'dark-blue',
```

Comment:
To be checked with the design team, but this background should probably stay, because the pattern has transparent buttons (and the text color is white 😅)